crates/sui-framework/sources/governance/validator.move

```diff
@@ -567,6 +567,42 @@ module sui::validator {
             // || self.metadata.net_address == other.metadata.net_address
             // || self.metadata.p2p_address == other.metadata.p2p_address
             || self.metadata.protocol_pubkey_bytes == other.metadata.protocol_pubkey_bytes
+            || self.metadata.network_pubkey_bytes == other.metadata.network_pubkey_bytes
```

Comment:
What about primary_address & worker_address, must they be unique?

Reply:
a node can't bind multiple services to one single socket. Although it's still possible for a very advanced player to place a proxy in the front and does all the routing, it's much easier to juts require unique addresses for all of them.

Reply:
That's a good point. While it could be problematic for the caller, I think that here we should only verify things that might break security or our protocols, but otherwise not limit the user. (Not being able to handle traffic because of a misconfigured address is similar to not being able to sign messages because I lost my keys, no?)